cmd/client/main.go

```diff
@@ -80,7 +83,6 @@ func main() {
 }
 
 func loadConfig() (*config, *client.Config) {
-
 	cfg := config{
```

Comment:
Load Consul config here. Change the way we connect to Mongo, as we will have Consul related stuff now in `cfg` also (not only Mongo ones).